apps/load-tests/backend/eslint.config.js

```diff
@@ -0,0 +1,24 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
+const eslintBaseConfig = require("../../../eslint.base.config");
+
+const customLanguageOptions = {
+  sourceType: "module",
+  parser: require("@typescript-eslint/parser"),
+  parserOptions: {
+    project: ["./tsconfig.json"],
+    ecmaVersion: "latest",
+    ecmaFeatures: {
+      jsx: true,
+      modules: true,
+    },
+  },
+};
+
+module.exports = [
+  ...eslintBaseConfig,
+  {
+    files: ["**/*.{ts,tsx}"],
```

Comment:
For backend configs we don't need JSX
```suggestion
      modules: true,
    },
  },
};

module.exports = [
  ...eslintBaseConfig,
  {
    files: ["**/*.ts"],
```

Reply:
This also applies to a number of other configs for packages that don't use React.

Reply:
removed for all relevant packages